### PR TITLE
Pause indicator

### PR DESCRIPTION
**Pull request by larrypo**
Adds an icon to help indicate breakpoints. Also puts the breakpoint color behind the breakpoint, error, and warning icons. 

This is for https://codedotorg.atlassian.net/browse/STAR-196

Screen Shot 2019-05-17 at 4 12 10 PM

### CODE REVIEW

File: apps/style/common.scss [hunk 1]
```diff
@@ -339,9 +339,14 @@ html[dir='rtl'] .editor-column {
   z-index: 3;
 }
 
+// By being defined before the rulthe pause icones for ace_warning and ace_error, the warning and error icons will take
```
Comment (islemaster) [inline apps/style/common.scss]: Looks like a little content got lost in this comment.

**Review by islemaster — CHANGES_REQUESTED** [review]
Hey Larry!  This looks great in text mode, but the pause icon isn't centered in block mode (at least on my machine - not sure how much this will vary from browser to browser).  I know the warning icon't isn't centered either, but while you're in there, can we see if we can get it looking great in both scenarios?

Peek 2019-05-20 14-07

File: apps/style/common.scss (hunk 1 above)
Comment (larrypo) [reply apps/style/common.scss]: Fixed.

**Review by islemaster — APPROVED** [review]
:tada: Looks awesome Larry!

**Comment by ryansloan** [comment]
Just saw that this is live in Staging! :tada: Woohoo!

**Pull request merged**